app/controllers/contacts_controller.rb

```diff
@@ -4,16 +4,18 @@
 class ContactsController < ApplicationController
   skip_verify_authorized
 
-  def new
-    @contact_form = ContactForm.new
-  end
+  def new; end
 
   def create
     contact_form = ContactForm.new(contact_form_params)
 
     ContactsMailer.with(contact_form.attributes.symbolize_keys).jira_email.deliver_later
 
-    redirect_to success_contacts_path
+    redirect_to success_contacts_path(bounce_location: params.dig('contact', 'bounce_location'))
+  end
+
+  def success
+    @bounce_location = params[:bounce_location]
```

Comment:
What is this? I don't see where success is being called.

Reply:
@justinlittman It's: https://github.com/sul-dlss/hungry-hungry-hippo/blob/25a7e5848d5e3509d87fe1783443980eab3ef057/app/controllers/contacts_controller.rb#L16 (route: https://github.com/sul-dlss/hungry-hungry-hippo/blob/25a7e5848d5e3509d87fe1783443980eab3ef057/config/routes.rb#L52)

Reply:
Duh.